codegen/datamodelgen.py

```diff
@@ -463,11 +481,19 @@ def _delete_generated_files(self):
             shutil.rmtree(Path(_SOLVER_DM_DOC_DIR))
 
 
-def generate(version, pyfluent_path, sessions: dict):
-    return DataModelGenerator(version, pyfluent_path, sessions).write_static_info()
+def generate(
```

Comment:
Don't forget to add "Parameters" and "Returns" sections with the appropriate descriptions.

Reply:
Done. Thank you.